nearcore/src/config.rs

```diff
@@ -376,6 +376,10 @@ fn default_cold_store_initial_migration_loop_sleep_duration() -> Duration {
     Duration::from_secs(30)
 }
 
+fn default_num_cold_store_read_threads() -> usize {
+    1
```

Comment:
I would set it higher, perhaps to 4? What do you think? What value did you use in tests?